programs/uxd/src/instructions/edit_controller.rs

```diff
@@ -121,6 +128,46 @@ pub(crate) fn handler(ctx: Context<EditController>, fields: &EditControllerField
             redeemable_global_supply_cap
         });
     }
+
+    // Optionally edit "outflow_limit_per_epoch_amount"
+    if let Some(outflow_limit_per_epoch_amount) = fields.outflow_limit_per_epoch_amount {
+        msg!(
+            "[edit_controller] outflow_limit_per_epoch_amount {}",
+            outflow_limit_per_epoch_amount
+        );
+        controller.outflow_limit_per_epoch_amount = outflow_limit_per_epoch_amount;
+        emit!(SetOutflowLimitPerEpochAmountEvent {
+            version: controller.version,
+            controller: ctx.accounts.controller.key(),
+            outflow_limit_per_epoch_amount
+        });
+    }
+
+    // Optionally edit "outflow_limit_per_epoch_bps"
+    if let Some(outflow_limit_per_epoch_bps) = fields.outflow_limit_per_epoch_bps {
+        msg!(
+            "[edit_controller] outflow_limit_per_epoch_bps {}",
+            outflow_limit_per_epoch_bps
+        );
+        controller.outflow_limit_per_epoch_bps = outflow_limit_per_epoch_bps;
```

Comment:
better with a extra checking to ensure the input bps <= BPS_POWER

Reply:
Good point ser! doing now 👌